packages/react-next/src/components/Link/Link.scss

```diff
@@ -24,7 +24,7 @@
     border-bottom: none;
   }
 
-  &.button {
+  &:not(.href) {
```

Comment:
Will the root ever have a class `href`?

Reply:
The way `compose` works right now is that it applies styling derived from props to classes that have the name of those props.

Reply:
Where does that happen? I feel like that's... Not obvious right now.